crawl-ref/source/dat/des/portals/bailey.des

```diff
@@ -1201,3 +1201,59 @@ ccc+cc.;;;;;;;;.cccc+c.---.t
     xx......xx
      xxxxxxxx
 ENDMAP
+
+# Prince Ribbit's fortress
+# If Prince Ribbit has already generated, his attendants have recovered the
+# wrong frog by mistake. Whoops!
+# Nine items, mostly consumables. Guaranteed broad axes from the vault guards.
+NAME:   bailey_axe_froggy_fort_hellmonk
+ORIENT: encompass
+TAGS:   bailey_axe no_item_gen no_monster_gen no_pool_fixup
+: item(dgn.loot_potions .. " / " .. dgn.loot_scrolls)
+: easy_axe_fighter(_G)
+MONS:   human ; hand axe ego:vorpal | war axe ego:vorpal . scale mail
+MONS:   vault guard ; broad axe . chain mail | plate armour
+KMONS:  4 = bullfrog
+KMONS:  5 = prince ribbit, blink frog / cane toad w:1
+KFEAT:  4 = shallow_water
+SHUFFLE: AB
+SUBST:  ; = W:3 ., B = <
+NSUBST: ! = 24 = 1... / 24 = 2... / *:.
```

Comment:
After playing, these good axe humans hit pretty hard for the early end of the depth; maybe only give them an ego axe half the time.

Reply:
A further follow up on this. These good axe humans are spookier than the frogs outside the throne room, even with the D:10 version of the same char. The pacing feels a bit weird on the map.